server/controllers/users.js

```diff
@@ -5,6 +5,7 @@ import emailLib from '../lib/email';
 import errors from '../lib/errors';
 import logger from '../lib/logger';
 import RateLimit, { ONE_HOUR_IN_SECONDS } from '../lib/rate-limit';
+import { verifyTwoFactorAuthenticatorCode } from '../lib/twoFactorAuth';
```

Comment:
We generally use kebab case for file names.
https://github.com/opencollective/opencollective/blob/master/rfcs/004-file-naming-convention.md

Reply:
Also let's generally keep names consistent, it can't be "twoFactorAuth" then "twoFactorAuthentication" elsewhere.